File: src/main/java/com/sica/backend/dto/VisitResponse.java
package com.sica.backend.dto;

import java.time.LocalDateTime;

public record VisitResponse(
    String id,
    String visitorName,
    LocalDateTime visitDatetime,
    String personVisited,
    String visitorPhoto,
    String status,
    String authorizedBy,
    LocalDateTime authorizedAt,
    String qrCodeBase64
) {}

File: src/main/kotlin/com/sica/backend/dto/VisitResponse.kt
package com.sica.backend.dto

import java.time.LocalDateTime

data class VisitResponse(
    val id: String,
    val visitorName: String,
    val visitDatetime: LocalDateTime,
    val personVisited: String,
    val visitorPhoto: String,
    val status: String,
    val authorizedBy: String? = null,
    val authorizedAt: LocalDateTime? = null,
    val qrCodeBase64: String? = null
)
